cirq/study/trial_result.py

```diff
@@ -227,15 +241,14 @@ def histogram(self,  # type: ignore
             results.
         """
         return self.multi_measurement_histogram(
-            keys=[key],
-            fold_func=lambda e: fold_func(e[0]))
+            keys=[key], fold_func=lambda e: fold_func(e.iloc[0]))
 
     def __repr__(self):
         return ('cirq.TrialResult(params={!r}, '
                 'repetitions={!r}, '
                 'measurements={!r})').format(self.params,
-                                             self.repetitions,
-                                             self.measurements)
+                                             self._num_repititions(),
```

Comment:
The repr would presumably change to use the pandas data instead of the derived data.

Reply:
Yes, in the subsequent PR which won't be backwards compatible.